common/app/model/content.scala

```diff
@@ -745,9 +745,10 @@ class Interactive(content: ApiContentWithMeta) extends Content(content) {
   override lazy val analyticsName = s"GFE:$section:$contentType:${id.substring(id.lastIndexOf("/") + 1)}"
 
   override lazy val metaData: Map[String, JsValue] = super.metaData + ("contentType" -> JsString(contentType))
-  override lazy val hideUi: Boolean = displayHint.contains("immersive") || body.exists{ b =>
+  override lazy val hideUi: Boolean = body.exists{ b =>
     Jsoup.parseBodyFragment(b).body().getElementsByClass("element-interactive").attr("data-interactive").contains("/visuals-blank-page/")
   }
+  override lazy val isImmersive: Boolean = displayHint.contains("immersive")
```

Comment:
Not sure whether this value belongs here on the Interactive class, or on the Content class.